test/unit/specs/vislib/lib/y_axis.js

```diff
@@ -210,128 +225,132 @@ define(function (require) {
         });
       });
 
-      describe('should not return a nice scale when defaultYExtents is true', function () {
+      describe('validate user defined values', function () {
+        var newDomain;
```

Comment:
The values of newDomain, min and max aren't reused across tests, but are redefined. Can you move these to be initally declared in each individual it()?